src/main/java/com/lewisd/maven/lint/rules/AbstractReduntantVersionRule.java

```diff
@@ -47,20 +47,20 @@ protected void checkForRedundantVersions(final MavenProject mavenProject,
         }
 
         String version = resolveVersion(modelObject, resolvedModelObject);
-        String inheritedVersion = modelUtil.getVersion(tryResolveObject(inheritedObject));
+        String inheritedVersion = getModelUtil().getVersion(tryResolveObject(inheritedObject));

```

Comment:
The Liskov doesn't say anything about "java's object orientation is used with accessors", it's about what they expose, and how they behave.  Whether classes use accessors or directly accesses protected fields has nothing to do with the Liskov substitution principle.  I agree that in 99% of cases, accessors should be used.
"If everything is opened" - I'm not talking about opening everything.  I've been very specific about using this for `final` utility classes only.

It seems we have different opinions on what constitutes "correct" on this, and I don't think either of us will be able to convince the other to change their opinion.  I really like the rest of the changes in this PR, and would be happy to merge them, but I still haven't changed my mind on this particular part, since it came up in the PR #5 in 2013.